<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>529</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>5</horstretch>
    <verstretch>5</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RAKABMAIN</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/C:/Users/Novin SOft/Desktop/New folder/Back_final.jpg</normaloff>:/images/C:/Users/Novin SOft/Desktop/New folder/Back_final.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="BG">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/image/src/Back_final.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_Btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/image/src/Play_But.png</normaloff>:/image/src/Play_But.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>700</width>
      <height>250</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Info_Btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/src/help_Btn.png</normaloff>:/src/help_Btn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>700</width>
      <height>250</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_Btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/src/Exit_but.png</normaloff>:/src/Exit_but.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>700</width>
      <height>250</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/src/Title.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
